package.json

```diff
@@ -6,23 +6,32 @@
   "directories": {
     "test": "test"
   },
+  "devDependencies": {
+    "@enigmampc/discovery-cli": "0.1.7",
+    "mocha": "^6.2.2"
+  },
   "dependencies": {
-    "@enigmampc/discovery-cli": "0.1.3",
     "@salad/client": "0.1.0",
     "@salad/operator": "0.1.0",
     "core-js": "^3.2.1",
     "debug": "^4.1.1",
     "dotenv": "^8.0.0",
     "enigma-js": "0.3.0",
-    "mocha": "^6.2.2",
     "openzeppelin-solidity": "^2.3.0",
     "supports-color": "^7.1.0"
   },
   "scripts": {
     "clean-run-operator": "yarn dc migrate && node ./operator/src/server.js -t -i",
     "run-operator": "node ./operator/src/server.js",
-    "test": "yarn dc migrate && mocha --exit ./test/api.test.js",
-    "dc": "node ./node_modules/@enigmampc/discovery-cli/src/index.js"
+    "dc": "discovery",
+    "configure": "scripts/configure.sh",
+    "configure-sw": "SGX_MODE=SW scripts/configure.sh",
+    "start": "yarn dc start",
+    "stop": "yarn dc stop",
+    "compile": "yarn dc compile",
+    "test": "yarn dc compile && mocha --exit ./test/api.test.js",
+    "migrate": "yarn dc compile && yarn dc migrate",
```

Comment:
The latest version of `discovery-cli` enforces that `compile` is always run before `migrate` so `yarn dc migrate` suffices here.

Reply:
:+1: